parameters.csv

```diff
@@ -7,38 +7,41 @@ checkStage,module list
 javaSuffix,Java-8-LTS
 javaVersion,Java/8-LTS
 hpoVersion,90
-gatkJar,gatk-package-4.2.4.2-local.jar
-gavinPlusJar,gavin-plus-1.5.0-RELEASE.jar
-picardJar,picard.jar
 vepDBVersion,92
 vepAssemblyVersion,GRCh37
 bcfToolsVersion,BCFtools/1.14-${toolChain_max}
 bedToolsVersion,BEDTools/2.30.0-${toolChain_max}
 bwaVersion,BWA/0.7.17-${toolChain_min}
 computeVersion,Molgenis-Compute/v19.01.1-${javaSuffix}
-caddVersion,CADD/v1.4-${toolChain_max}
+caddVersion,CADD/v1.3-${toolChain_max}
 convadingVersion,1.2.1-${toolChain_min}
 fastqcVersion,FastQC/0.11.9-${javaSuffix}
 gatkVersion,GATK/4.2.4.1-${javaSuffix}
+gatkJar,gatk-package-4.2.4.2-local.jar
```

Comment:
Version 4.2.4.2 is a mismatch with gatkVersion,GATK/4.2.4.1-${javaSuffix}

Reply:
We should install a symlink without version number to file with version number using the EasyConfig:
`gatk-package-local.jar -> gatk-package-4.2.4.2-local.jar`
Then you do not need to specify the version again and can drop the gatkJar variable. Simply a `module load GATK/4.2.4.1-....` is then enough to do something like `java -jar ${EBROOTGATK}/gatk-package-local.jar`